etc/kayobe/inventory/groups

```diff
@@ -87,6 +93,7 @@ network
 monitoring
 storage
 compute
+infra-vms
```

Comment:
This could be a breaking change on some systems. Could we instead just add gitlab-runners and github-runners?